src/sentry/relocation/models/relocationtransfer.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import annotations
+
+from django.db import models
+from django.utils import timezone
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import control_silo_model, region_silo_model
+from sentry.db.models.base import DefaultFieldsModel
+from sentry.db.models.fields.uuid import UUIDField
+
+
+class RelocationTransferState(models.TextChoices):
+    Request = "request"
+    Reply = "reply"
+
+
+class BaseRelocationTransfer(DefaultFieldsModel):
+    """
+    Base class for control + region relocation transfer models
+
+    Relocation transfers are used to record a retriable
+    state of a regional transfer for relocation data.
+    These models replace outbox based transfers.
+    """
+
+    __relocation_scope__ = RelocationScope.Excluded
+
+    relocation_uuid = UUIDField()
+    org_slug = models.CharField(null=False)
+    requesting_region = models.CharField(null=False)
+    exporting_region = models.CharField(null=False)
+    state = models.CharField(
+        choices=RelocationTransferState, default=RelocationTransferState.Request
+    )
+    scheduled_for = models.DateTimeField(null=True, default=timezone.now)
+    public_key = models.BinaryField(null=True)
```

Comment:
What's the purpose of storing the public key here on this model? It doesn't look like something we previously persisted on the outbox.

Reply:
We currently store the public key in the export request outboxes https://github.com/getsentry/sentry/blob/3a889656e4e46d5712280f5afd24f63810286071/src/sentry/relocation/services/relocation_export/impl.py#L162-L170

I think nullable columns are a better solution than adding another JSON blob field.

Reply:
Ahhhh okay this is the fundamental piece I was missing, thanks for linking!

Would it make sense to only add this to the control variant of the model then, or are you planning to store this on the region side as well?

Reply:
> Would it make sense to only add this to the control variant of the model then, or are you planning to store this on the region side as well?

Yeah that would work.